chapter05/dtj/ex3

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+print "How long is a string? ";
+chomp( my $width = <STDIN> );
+
+print "What do you want printed? ";
+chomp( my @words = <STDIN> );
+
+print "1234567890" x ($width/10+1), "\n";
```

Comment:
All looks good, only minor quibble would be formatting here. 10+1 still jumps out at me even though I know about operator precedence.